packages/rum/src/domain/startRecordBridge.ts

```diff
@@ -0,0 +1,17 @@
+import { getEventBridge } from '@datadog/browser-core'
+import type { ViewContexts } from '@datadog/browser-rum-core'
+import type { BrowserRecord } from '../types'
+
+export function startRecordBridge(viewContexts: ViewContexts) {
+  const bridge = getEventBridge<'record', BrowserRecord>()!
+
+  return {
+    addRecord: (record: BrowserRecord) => {
+      const view = viewContexts.findView()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should not we get the view corresponding to the record timestamp?